react-client/src/components/Map.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import GoogleMapReact from 'google-map-react';
+import { Marker } from 'react-google-maps';
+import $ from 'jquery';
+
+
+class Map extends React.Component {
+ constructor(props){
+      super(props)
+      
+      this.state={
+            specialties:[],
+            markerPosition:null,
+            isMarkerShown:false
+        }
+
+      this.onMapClick=this.onMapClick.bind(this)
+      this.showNearest=this.showNearest.bind(this);
+  }
+  showNearest(){
+    console.log('hi')
+      if(this.state.markerPosition && this.props.specialty){
+            console.log(this.state.markerPosition.lat,this.state.markerPosition.lng)
+
+      
+      
+      var that=this
```

Comment:
All of this should be indented further right than the beginning of the if statement. Also a lot of unnecessary blank lines.